hive_metastore_client/builders/partition_builder.py

```diff
@@ -0,0 +1,56 @@
+"""PartitionBuilder."""
+from thrift_files.libraries.thrift_hive_metastore_client.ttypes import Partition, StorageDescriptor, PrincipalPrivilegeSet  # type: ignore # noqa: E501
+from typing import List, Mapping
+
+
+class PartitionBuilder:
+    """Builds thrift Partition object."""
+
+    def __init__(
+        self,
+        values: List[str],
+        db_name: str,
+        table_name: str,
+        sd: StorageDescriptor = None,
+        create_time: int = None,
+        last_access_time: int = None,
+        parameters: Mapping[str, str] = None,
+        privileges: PrincipalPrivilegeSet = None,
+        cat_name: str = None,
+    ) -> None:
+        """
+        Constructor.
+
+        :param values: list of string values of table partition keys
```

Comment:
You don't need to specify types here once we already do that in the method's signature.
What about:
```suggestion
        :param values: list partition keys names
```